src/create-airseeker.ts

```diff
@@ -0,0 +1,187 @@
+import { Choice, PromptObject } from 'prompts';
```

Comment:
Where is this script used? Should this be `create-airseeker-config.ts`? Should it be exposed in `package.json`?

Reply:
yeah I will rename it to create-airseeker-config  and add a command in package.json.